### PR TITLE
Update fabs to std::abs per SAM #1223

### PR DESCRIPTION
**Pull request by sjanzou**
Other repos will have similar pull request coming!

### CODE REVIEW

**Comment by sjanzou** [comment]
Address SAM issue 1223 in the lk repo https://github.com/NREL/SAM/issues/1223

**Review by tyneises — APPROVED** [review]
This looks good. My main concern was instances of abs() without the std:: prefix. Do you also recommend changing all the fabs() in the code base to std::abs, as you do here?

File: src/stdlib.cpp [hunk 1]
```diff
@@ -35,8 +35,7 @@
 #include <future>
 #include <chrono>
 
-#include <cmath>
-#include <float.h>
+//#include <float.h>
```
Comment (brtietz) [inline src/stdlib.cpp:38]: Please remove commented out code

**Review by brtietz — COMMENTED** [review]
Changes look fine, but please avoid commented out code when possible

**Pull request merged**